src/app/templates/show-header.tpl

```diff
@@ -0,0 +1,31 @@
+<ul class="panel-group nav-hor left" id="accordion" role="tablist" aria-multiselectable="true">
+    <li class="panel">
+        <a class="source" role="button" data-toggle="collapse" data-parent="#accordion" href="#collapseOne" aria-expanded="true" aria-controls="collapseOne">
+            <%= i18n.__('Show Info') %>
+        </a>
+        <div id="collapseOne" class="panel-collapse collapse in" role="tabpanel" aria-labelledby="headingOne">
+        </div>
+    </li>
+    <%_.each(torrents, function(episodes, season) { %>
+    <li class="panel">
+        <a class="source" role="button" data-toggle="collapse" data-parent="#accordion"
+           href="#show-detail-collapse-<%= season %>"
+           aria-expanded="false" aria-controls="show-detail-collapse-<%= season %>">
+            <%= i18n.__('Season ' + season) %>
```

Comment:
then i18n.__('Season %s', season), but no +. You're screwing the entire point of translation with a +, because the string is modified before being sent to i18n, which will then try to translate 'Season 18' instead of 'Season' and a number.